docs/cookbook/admin_panel/index.md

```diff
@@ -0,0 +1,130 @@
+# How to customize your admin panel
+
+## How to customize the sidebar logo
+
+To customize the sidebar logo, you need to configure the new template to use using the following configuration.
```

Comment:
`to use using` -- English not englishing. :slightly_smiling_face: 
 
for example,
```suggestion
To customize the sidebar logo, you need to set new logo template at `sylius_admin.common.component.sidebar.logo` twig hook.
```